eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -355,6 +355,18 @@
           "new": "parameter <T> reactor.core.publisher.Mono<T> com.azure.spring.data.cosmos.core.ReactiveCosmosTemplate::insert(java.lang.String, ===T===, com.azure.cosmos.models.PartitionKey)",
           "parameterIndex": "1",
           "justification": "To support mono method chaining, without explicit typcast for upper bounded generics"
+        },
+        {
+          "code": "java.annotation.attributeRemoved",
+          "annotationType": "com.azure.core.annotation.ServiceClient",
+          "new": "class com.azure.data.appconfiguration.ConfigurationClient",
+          "justification": "These are SDK metadata annotations and don't affect runtime behavior."
+        },
+        {
+          "code": "java.annotation.attributeRemoved",
+          "annotationType": "com.azure.core.annotation.ServiceClient",
+          "new": "class com.azure.data.appconfiguration.ConfigurationAsyncClient",
```

Comment:
If we add the service interface back to the annotation, we can remove the suppressions here.